range.c

```diff
@@ -554,6 +553,33 @@ range_step(int argc, VALUE *argv, VALUE range)
     return range;
 }
 
+/*
+ *  call-seq:
+ *    %(n) {|element| ... } -> self
+ *    %(n)                  -> enumerator
+ *
+ *  Iterates over the elements of +self+.
+ *
+ *  With a block given and no argument,
```

Comment:
Examples show an argument is used, and function definition shows an argument is required.  I would just remove ` and no argument`.

Reply:
Done.